### PR TITLE
cpufreq-info: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/cpufreq-info.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show cpufreq kernel information for all CPUs:
```
Comment (sbrl) [inline pages/linux/cpufreq-info.md]: Perhaps `cpu frequency` would read better then `cpufreq`?

File: pages/linux/cpufreq-info.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+`cpufreq-info`
+
+- Show cpufreq kernel information for specified CPU:
+
+`cpufreq-info -c {{cpu_number}}`
+
+- Show current CPU work frequency in human-readable format, according to the cpufreq core:
```
Comment (sbrl) [inline pages/linux/cpufreq-info.md]: Perhaps `in a human-readable format`..... would read better?

Also, what is mean by `cpufreq core`?

File: pages/linux/cpufreq-info.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show cpufreq kernel information for all CPUs:
+
+`cpufreq-info`
+
+- Show cpufreq kernel information for specified CPU:
```
Comment (sbrl) [inline pages/linux/cpufreq-info.md]: Perhaps ...`for the specified`... would read better?

File: pages/linux/cpufreq-info.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+- Show the minimum and maximum CPU frequency allowed:
+
+`cpufreq-info -l`
+
+- Show the currently used cpufreq policy:
+
+`cpufreq-info -p`
+
+- Show available cpufreq policy:
```
Comment (sbrl) [inline pages/linux/cpufreq-info.md]: Perhaps ...`policies` would read better here?

File: pages/linux/cpufreq-info.md (hunk 2 above)
Comment (jsonbruce) [reply pages/linux/cpufreq-info.md]: Actually,  `cpufreq` is the infrastructure module of implementing CPU frequency scaling in Linux kernel.
So we'll use the kernel module `cpufreq` when invoking `cpufreq-info -f` to retrieve CPU frequency, which differs from `cpufreq-info -w`  using the hardware information.

Comment (sbrl) [reply pages/linux/cpufreq-info.md]: Ah, I see - thanks 😺

File: pages/linux/cpufreq-info.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show CPU frequency kernel information for all CPUs:
```
Comment (agnivade) [inline pages/linux/cpufreq-info.md]: In the description, it is mentioned "CPU frequency information", but here you write "CPU frequency kernel information". Why is the word "kernel" added ?

File: pages/linux/cpufreq-info.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+`cpufreq-info`
+
+- Show CPU frequency kernel information for the specified CPU:
+
+`cpufreq-info -c {{cpu_number}}`
+
+- Show current CPU work frequency in a human-readable format, according to the kernel module:
```
Comment (agnivade) [inline pages/linux/cpufreq-info.md]: which kernel module ?

File: pages/linux/cpufreq-info.md (hunk 5 above)
Comment (jsonbruce) [reply pages/linux/cpufreq-info.md]: It's a mistake. I;ve fixed it.

File: pages/linux/cpufreq-info.md (hunk 6 above)
Comment (jsonbruce) [reply pages/linux/cpufreq-info.md]: The `cpufreq` kernel module. I've updated  the documentation.

**Pull request merged**